cmd/kube-apiserver/app/options/options.go

```diff
@@ -173,7 +171,7 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.IPNetVar(&s.ServiceClusterIPRange, "service-cluster-ip-range", s.ServiceClusterIPRange, ""+
 		"A CIDR notation IP range from which to assign service cluster IPs. This must not "+
-		"overlap with any IP ranges assigned to nodes for pods.")
+		"overlap with any IP ranges assigned to nodes for pods. [default='"+master.DefaultServiceIPCIDR+"'].")
```

Comment:
Cobra will automatically print the default value for the flag help text if such a default exists. Would it be possible to set `s.ServiceClusterIPRange` to `10.0.0.0/24` in `NewServerRunOptions()` above? Would doing so break anything?

Reply:
done.